addon/validators/presence.js

```diff
@@ -12,7 +12,6 @@ export default function validatePresence(options) {
     if (typeof result === 'boolean' || typeof result === 'string') {
       return result;
     } else {
-
       // We flipped the meaning behind `present` and `blank` so switch the two
       if (result.type === 'present') {
```

Comment:
That's weird. This just flips the two error messages programatically... There shouldnt be a need for this anymore. 

I think the tests fail because its checking for the `type` not the actual error message itself. Can you verify?